docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-tutorials/tutorial-troubleshooting-a-failed-deployment.md

```diff
@@ -4,42 +4,118 @@ description: Learn how to troubleshoot a failed deployment
 template: troubleshooting-guide-template
 ---
 
-If a deployment fails and there are no errors in the deploy or build output, do the following:
+A deployment fails, and there are no errors in the deploy or build output. Based on the step the deployment fails at, do the following:
 
 
-## 1. Check deploy logs
+## Deployment fails at the Build_Push_if_not_exist step
 
-Check the build logs via Log groups as follows. Filter the log groups by `codebuild`. If the deployment fails at the `Deploy_Scheduler` step, after checking the `codebuild` logs, filter by `jenkins` and check the logs.
+If a deployment fails at the `Build_Push_if_not_exist`, do the following.
 
-{% include searching-by-logs.md %} <!-- To edit, see /_includes/searching-by-logs.md -->
+### 1. Check deploy logs
 
+Check the build logs via Log groups as follows. Filter the log groups by `codebuild`.
 
+{% include searching-by-logs.md %} <!-- To edit, see /_includes/searching-by-logs.md -->
 
-## 2. Check multiple log groups at once
+### 2. Check deploy logs in multiple log groups at once
 
-Check multiple log groups via Logs Insights as follows. Select the log groups containing `codebuild`. If the deployment fails at the `Deploy_Scheduler` step, also select the groups containing `jenkins`.
+Check multiple log groups via Logs Insights as follows. Select the log groups containing `codebuild`.
 
 {% include searching-by-multiple-log-groups.md %} <!-- To edit, see /_includes/searching-by-multiple-log-groups.md -->
 
-## 3. Check the status of services and tasks
+### 3. Check Build_Push_if_not_exist step execution details
 
-If the deployment fails at the `Deploy_Spryker_services` step, check the failed services and tasks for errors as follows:
+1. On the page of the deployment in the `Build_Push_if_not_exist` step, select **Details**.
 
-{% include checking-the-status-of-ecs-services-and-tasks.md %} <!-- To edit, see /_includes/checking-the-status-of-ecs-services-and-tasks.md -->
+![execution_details]
 
+2. In the *Action execution failed* window that opens, select **Link to execution details**.
```

Comment:
window name should be in bold
https://developers.google.com/style/ui-elements?hl=en#formatting